README.md

```diff
@@ -18,7 +18,11 @@ The web server repository can be found [here](https://github.com/marcelorcorrea/
 
 Feel free to fork or contribute this project.
 
-Download in [Play Store](https://play.google.com/store/apps/details?id=org.falaeapp.falae)
+[![Slack channel](https://img.shields.io/badge/Chat-Slack-blue.svg)](https://falaeapp.slack.com)
+[![Language Kotlin](https://img.shields.io/badge/Language-Kotlin-blue.svg)](https://kotlinlang.org/)
+[![License MIT](https://img.shields.io/badge/Licence-Mit-blue.svg)](LICENSE)
```

Comment:
Just for checking, is there a badge link with `License` with 's', and `MIT` in capital letters?

Reply:
Yes. There is. This is not static, if you change the name of svg file, you can change it at all.